frontends/web/src/routes/buy/utils.ts

```diff
@@ -63,4 +63,33 @@ export function getFormattedName(name: Omit<Info, 'region'>) {
   case 'pocket':
     return 'Pocket';
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Finds the region name from locale.
+ * Example: `en-DE` will return `DE`
+ */
+export const getRegionNameFromLocale = (nativeLocale?: string): string => {
```

Comment:
Hmm.. sure we can move them into a file together. Since config.js is a JS file, I think it'd be better to make a ts file that contains these utility functions (as converting the whole config.js to ts will be a bit too broad for this PR). 

I'll create a utils.ts inside the i18n dir, which contains the util functions (i18nextFormat, localeMainLanguage, and then also getRegionNameFromLocale).